scipy/sparse/linalg/isolve/iterative.py

```diff
@@ -430,7 +430,7 @@ def cgs(A, b, x0=None, tol=1e-5, maxiter=None, M=None, callback=None, atol=None)
 
 @non_reentrant()
 def gmres(A, b, x0=None, tol=1e-5, restart=None, maxiter=None, M=None, callback=None,
-          restrt=None, atol=None, callback_type=None):
+          restrt=None, atol=None, callback_type=None, orth=None):
```

Comment:
> I left other comments (still cosmetic, if no-one reviews the maths it might take some time for me to get on that as I would need to read the literature, etc.). Thanks for the update.

Thank you for your suggestion. I'll try to modify it

Reply:
FYI, if you agree with suggestions, go to the file panel and you can click on the `Commit suggestion` button and there is a group option to make a single commit with multiple suggestions.

Reply:
> FYI, if you agree with suggestions, go to the file panel and you can click on the `Commit suggestion` button and there is a group option to make a single commit with multiple suggestions.

Thank you so much, it is very convenient. I would like to ask how the updated code on GitHub keeps paces with my local machine (I used to change codes in local firstly, and then submit to GitHub), thanks.